src/ipbb/cli/repo.py

```diff
@@ -41,11 +41,12 @@ def add(env):
 @click.argument('repo')
 @click.option('-b', '--branch', default=None, help='Git branch or tag to clone')
 @click.option('-d', '--dest', default=None, help="Destination directory")
+@click.option('-r', '--revision', default=None, help='Git revision ID to clone')
```

Comment:
Style comment: I would prefer to have the `dest` option last.

Reply:
Sure. I was going for alphabetical, but I don't feel very strongly about the alphabet ;-)

Reply:
Ok. I changed the order, both of the options and of the corresponding arguments.